bindata/network/infra-dpu-cni/infra-dpu-cni.yaml

```diff
@@ -0,0 +1,163 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: infra-dpu-cni-script
+  namespace: openshift-infra-dpu-cni
+  annotations:
+    kubernetes.io/description: |
+      This is a script that serves as the CNI on DPU. Only host network supported, so returns error to all calls.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+data:
+  infra-dpu-cni.sh: |-
+    #!/bin/bash
+    set -e
+
+    exec 3>&1 # make stdout available as fd 3 for the result
```

Comment:
Why? Is this leftover from an earlier attempt to make fd 1 _not_ be stdout?

note that you redirect the ADD output to 3, but leave the VERSION output on 1...